pset-functions.js

```diff
@@ -115,6 +171,60 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+// function isPalindrome(word) {
+//   let i = 0
+//   let letters = word.split('')
+//   console.log(letters)
+//   for (let i = 0; i < letters.length; i++) {
+//     let currentLetter = letters[i]
+//     let lastLetter = letters[(letters.length - 1 - i)]
+//     console.log(i)
+//     console.log(letters.length - i)
+//     if (currentLetter === lastLetter) {
+//       return true
+//       console.log(currentLetter)
+//     }
+//     else {
+//       return false
+//       console.log(currentLetter)
+//       console.log(lastLetter)
+//     }
+//   }
+// }
+
+function isPalindrome(word) {
+  let letters = word.split('')
+  let lettersReverse = letters.slice(0).reverse()
+  console.log(lettersReverse)
+  console.log(letters)
+  for (let i = 0; i < letters.length; i++) {
+    if (letters[i] !== lettersReverse[i]) {
+      return false
+    } else {
+      }
+  }
+  return true
+}
+
+// const palindrome = isPalindrome()
+let palindrome = isPalindrome('hello')
+console.log(palindrome)
+
+// figure out how to reverse array. see if array === reversed array
+
+
+function pallyPalindrome(word) {
+  let letters = word.split('')
+  let lettersReverse = letters.reverse()
+  let wordReverse = lettersReverse.join('')
+    return word === wordReverse
```

Comment:
Ah, looks like you solved the palindrome problem two different ways; way to refactor and simplify your code!